build/.cmake/api/v1/query/client-neocmake/query.json

```diff
@@ -0,0 +1 @@
+{"requests":[{"kind":"codemodel","version":[{"major":2,"minor":8}]},{"kind":"configureLog","version":[{"major":1,"minor":0}]},{"kind":"cache","version":[{"major":2,"minor":0}]},{"kind":"cmakeFiles","version":[{"major":1,"minor":1}]},{"kind":"toolchains","version":[{"major":1,"minor":0}]}]}
```

Comment:
remember to cleanup

Reply:
Fixed in [4d21744](https://github.com/RediSearch/RediSearch/commit/4d217443cdf580e353b3b14444604153a5ac8f8c)